### PR TITLE
Homework 2 30.07

### PR DESCRIPTION
**Pull request by DanaMan**

### CODE REVIEW

File: java-grupa8/homework-2/src/main.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by Dana Man on 7/30/17.
+ */
+public class main {
```
Comment (andrei-hegedus) [inline java-grupa8/homework-2/src/main.java]: This class is never used. Please delete it.

File: java-grupa8/homework-2/src/sort.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/**
+ Implements a bubble sort algorithm to sort an array of SalesRepresentative objects in descending order.
+ The sorting should be done based on the amount of revenue generated via sales.
+ Each rep has a number of sales, and a quota / sale.
+ */
+public class sort {
```
Comment (andrei-hegedus) [inline java-grupa8/homework-2/src/sort.java]: All classes must start with capital letters. Please check the naming conventions.

File: java-grupa8/homework-2/src/sort.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    public static void main(String[] args) {
+            String[] nameRepr = {"Ion Dumbraveanu", "Alina Iriminescu", "Ovidiu Zaharescu", "Nicoleae Popovici", "Andeea Maxim"};
+            int[] numberSales = {10, 30, 20, 51, 20};
+            int[] quotaSales = {500, 250, 400, 100, 450};
+            int[] revenue = new int[5];
+            for (int a = 0; a < nameRepr.length; a++) {
+                revenue[a] = numberSales[a] * quotaSales[a];
+            }
+            int tempValue = 0;
```
Comment (andrei-hegedus) [inline java-grupa8/homework-2/src/sort.java]: As the requirements state: Main class and algorithm classes separated.
The Bubble sort algorithm should be extracted in a separate method or class.

File: java-grupa8/homework-2/src/main.java (hunk 1 above)
Comment (DanaMan) [reply java-grupa8/homework-2/src/main.java]: Done

File: java-grupa8/homework-2/src/sort.java (hunk 2 above)
Comment (DanaMan) [reply java-grupa8/homework-2/src/sort.java]: Done

File: java-grupa8/homework-2/src/sort.java (hunk 3 above)
Comment (DanaMan) [reply java-grupa8/homework-2/src/sort.java]: Done

**Pull request merged**